### PR TITLE
Apply styles to UI

### PR DESCRIPTION
**Pull request by sntakirutimana72**
## REQUIREMENTS
- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- Use the information detailed in Zeplin to style the website.
- Your final design should match Zeplin's design.

## MILESTONE

- Applied different styles to UI features.
- Fixed linter errors.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -29,12 +29,22 @@ const Navbar = () => {
...
-      <button type="button" onClick={toggleUser} className={styles.Profile}>
+      <Link to="/" className={styles.Home}>Bookstore CMS</Link>
+      <div>
+        <ul className={styles.UlLinks}>
+          <li>
+            <Link to="/" className={styles.Link}>BOOKS</Link>
+          </li>
+          <li>
+            <Link to="/categories" className={styles.Link}>CATEGORIES</Link>
+          </li>
```
Comment (ShoiraTa) [inline src/components/Navbar.js:40]: - [x] Based on the requirements for this milestone, styling should be Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Kindly make sure that the active navbar button has brighter color and non-active greyish

Your design  | Zeplin design
------------- | -------------
Screenshot 2022-07-14 at 18 12 42 | Screenshot 2022-07-15 at 16 30 51

File: src/components/Book.js [hunk 2]
```diff
@@ -82,13 +90,14 @@ const Book = ({ book }) => {
       </div>
       <div className={css.BookActions}>
         <ActionButton title="Comments" />
+        <div className={css.LineSep} />
         <ActionButton trigger={onDelete} title="Remove" />
+        <div className={css.LineSep} />
         <ActionButton title="Edit" />
       </div>
```
Comment (ShoiraTa) [inline src/components/Book.js:97]: - [x] Action buttons should be aligned to the left without padding:

Your design  | Zeplin design
------------- | -------------
Screenshot 2022-07-15 at 16 40 04 | Screenshot 2022-07-15 at 16 41 17

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @sntakirutimana72 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

**First let me to point out:**

- Great commit history! ✔️ 
- You passed all the Linters! ✔️ 
- Professional pull request! ✔️ 

Keep on coding, good job so far! :clap:

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — COMMENTED** [review]
Hi @sntakirutimana72, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**